corehq/ex-submodules/couchforms/models.py

```diff
@@ -383,14 +383,14 @@ def unarchive(self, user_id=None):
             archive_stub.archive_history_updated()
             xform_unarchived.send(sender="couchforms", xform=self)
 
-    def publish_archive_action_to_kafka(self, user_id, archive):
+    def publish_archive_action_to_kafka(self, user_id, archive, rebuild_cases=True):
         from couchforms.models import UnfinishedArchiveStub
         from corehq.form_processor.submission_process_tracker import unfinished_archive
         # Delete the original stub
         UnfinishedArchiveStub.objects.filter(xform_id=self.form_id).all().delete()
         with unfinished_archive(instance=self, user_id=user_id, archive=archive):
             if archive:
-                xform_archived.send(sender="couchforms", xform=self)
+                xform_archived.send(sender="couchforms", xform=self, rebuild_cases=rebuild_cases)
             else:
                 xform_unarchived.send(sender="couchforms", xform=self)
```

Comment:
Just noticed this now and I realise it's not required in this PR but it would make this more complete if you implemented the same skip options for un-archiving.

Reply:
sure, will take it up in a follow up PR

Reply:
included this as well.